src/Cake.Core/IO/Globber.cs

```diff
@@ -56,6 +56,19 @@ public Globber(IFileSystem fileSystem, ICakeEnvironment environment)
         /// </returns>
         public IEnumerable<Path> Match(string pattern)
         {
+            return Match(pattern, null);
+        }
+
+        /// <summary>
+        /// Returns <see cref="Path" /> instances matching the specified pattern.
+        /// </summary>
+        /// <param name="pattern">The pattern to match.</param>
+        /// <param name="predicate">The predicate used to filter directories based on file system information.</param>
+        /// <returns>
+        ///   <see cref="Path" /> instances matching the specified pattern.
+        /// </returns>
+        public IEnumerable<Path> Match(string pattern, Func<IFileSystemInfo, bool> predicate)
```

Comment:
Ok, I understand. Maybe we should remove the other overload then, and introduce the predicate less method as an extension method to keep the interface lean.